flatpak/manifest_tool/tests/conftest.py

```
@@ -10,9 +10,9 @@
 if str(PACKAGE_ROOT) not in sys.path:
     sys.path.insert(0, str(PACKAGE_ROOT))
 
-import pytest
+import pytest  # noqa: E402
```

Ruff (0.13.1)

Unused `noqa` directive (non-enabled: `E402`)

Remove unused `noqa` directive

(RUF100)